testing/demo/pkg/setup-env/setup-env.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func main() {
```

Comment:
do we still want to add a copy of .env in the root? I think it's good to separate different envs. @cmwaters @rootulp

Reply:
Are there environment variables that we need set by the repo root that aren't present in the `solidity-ibc-eureka/.env`?

Reply:
yeah @cmwaters mentioned that he'll need env for his service

Reply:
I'm not strongly opinionated on where where we store the .envs or how many .env files we have. 

I'm fine with one root level .env that defines defaults for all the env variables the repo uses.